terraform/xmtp-validation-service/main.tf

```diff
@@ -0,0 +1,103 @@
+locals {
+  name              = "xmtp-validation-service"
+  service_port      = 50051
+  health_check_port = 50052
+  health_check_path = "/health"
+}
+
+module "service_sg" {
+  source  = "terraform-aws-modules/security-group/aws"
+  version = "5.2.0"
+
+  name        = "validation"
+  description = "Security group for the validation service allowing inbound traffic from inside the VPC"
+  vpc_id      = var.vpc_id
+
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = local.service_port
+      to_port     = local.service_port
+      protocol    = "tcp"
+      description = "Service port"
+      # Only allow access from private subnets
+      cidr_blocks = join(",", var.allowed_ingress_cidr_blocks)
+    },
+  ]
+  egress_with_cidr_blocks = [
```

Comment:
this service has no egress. Right?

Reply:
I think the validation service is calling out to the chain RPC URL's